LibAddonMenu-2.0/controls/submenu.lua

```diff
@@ -138,7 +146,7 @@ function LAMCreateControl.submenu(parent, submenuData, controlName)
     --figure out the cool animation later...
     control.animation = am:CreateTimeline()
     local animation = control.animation
-    animation:SetPlaybackType(ANIMATION_SIZE, 0) --2nd arg = loop count
+    animation:SetPlaybackType(ANIMATION_PLAYBACK_ONE_SHOT, 0) --2nd arg = loop count
```

Comment:
This is also a functional change, and one I don't feel very confident in - I just know that before, the behavior was undefined: ANIMATION_SIZE = 5 is outside the range of AnimationPlayback (-1 to 2).

I can't be sure whether you meant ANIMATION_PLAYBACK_LOOP or ANIMATION_PLAYBACK_PING_PONG, though.